src/components/ModalPayment/SubmissionButton.tsx

```diff
@@ -7,18 +7,22 @@ type Props = {
 
 const SubmissionButton = ({ canSubmit }: Props) => {
   const context = useContext(Context);
+  var [submittable, setSubmittable] = useState(false);
 
   const handleSubmit = (evt: { preventDefault: () => void }) => {
     evt.preventDefault();
     context.onCreateNonce();
+    setSubmittable(false);
   };
 
+  submittable = canSubmit;
+
   return (
     <button
       type="button"
       className={'modalButton--filled'}
       onClick={handleSubmit}
-      disabled={canSubmit === false}
+      disabled={!submittable}
```

Comment:
Does this also work for Enter to submit? Do we even support that?

Reply:
It doesn't look like we support that atm. Thats a good idea for a feature though, I can try adding it in another PR